conf/redis_cache.conf.example

```diff
@@ -0,0 +1,830 @@
+# Redis configuration file example
```

Comment:
Do we really need to have all the comments in there.

I think just having the relevant configuration with a comment of each parameter would be enough.